src/CardContainer/CardContainer.js

```diff
@@ -14,6 +14,7 @@ const CardContainer = ({movieData}) => {
   })
 
   return (
+    !movies.length ? <main className="loading">HEY</main> : 
```

Comment:
If there are no movies, add .loading to the classes to main - I'm not sure we're still using this.